ang/civicase/ActivityCard--Menu.html

```diff
@@ -1,8 +1,8 @@
-<div class="btn-group btn-group-sm activity-card-menu">
+<div class="btn-group btn-group-sm civicase__3-dot-menu">
```

Comment:
`civicase__3-dot-menu` does not sound good honestly. What about `civicase__activity-menu`